examples/medplum-chat-demo/data/example/example-data.json

```diff
@@ -230,6 +230,49 @@
         "subject": { "reference": "urn:uuid:ec42014b-63af-407e-ba31-c5b080c527a4" },
         "payload": [{ "contentString": "I can do it, but it will have to be quick and he must be on time." }]
       }
+    },
+    {
+      "fullUrl": "urn:uuid:23b65226-f1fd-424b-90c7-af2914dd6eba",
+      "request": { "method": "POST", "url": "ValueSet" },
+      "resource": {
+        "resourceType": "ValueSet",
+        "status": "active",
+        "url": "https://example.org/thread-categories",
+        "name": "thread-category-codes",
+        "title": "Thread Category Codes",
+        "description": "These codes identify categories to be help organize threads.",
+        "compose": {
+          "include": [
```

Comment:
Since these are all based on specialties, we should probably find the real codes (e.g. snomed) for them